documentation/debugging-coredump.md

```diff
@@ -7,6 +7,8 @@ Dumps created with gdb or gcore do not have all the managed state so various SOS
 
 ### Getting symbols ###
 
+Because SOS now has symbol download support, this step can be skipped if the correct version of the "host" program is available on the machine loading the dump. Usually the host program is "dotnet" but each .NET Core SDK has a different version.
```

Comment:
Replace 'correct' with 'matching'? Its a little ambiguous what version of the host is the 'correct' one.